CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [Code of Conduct](CODE_OF_CONDUCT.md).
```

Comment:
@SamBarker this assumes your code of conduct file will land in this repo first.

Reply:
Thats now done.